pyuoi/linear_model/base.py

```diff
@@ -204,6 +204,12 @@ def fit(self, X, y, stratify=None, verbose=False):
 
         # iterate over bootstraps
         for bootstrap in range(chunk_size):
+            # reset the coef between bootstraps
+            if hasattr(self.selection_lm, 'coef_'):
+                self.selection_lm.coef_ = np.zeros(
```

Comment:
```suggestion
                self.selection_lm.coef_ = np.zeros_like(
                    self.selection_lm.coef_,
                    dtype=X.dtype,
                    order='F')
```

I would use zeros_like